### ISSUE TITLE
GSS - conditional mixin

### ISSUE THREAD

**Issue by slavap** [google/closure-stylesheets#16]
There should be some way to process mixin parameters in conditional way, e.g.
@defmixin test(AAA) {
  @if (AAA == 5) {...}
  @else {...}
}

As simpler solution, may be selectFrom should be extended to be like Oracle decode() http://docs.oracle.com/cd/B19306_01/server.102/b14200/functions040.htm

**Comment by iflan** [comment]
Yes, we would definitely like to have this as a feature. It is just not built yet. If you're interested in developing it, let me know.
